packages/plugin-terraform/src/index.ts

```diff
@@ -100,29 +102,38 @@ export const plugin: PluginFunction<PluginConfig> = (
         const schema = tfg.resource('amplience_content_type_schema', name, {
           body: fn(
             'file',
-            `\${path.module}/schemas/${paramCase(node.name.value)}.json`
+            `\${path.module}/schemas/${paramCase(node.name.value)}${
+              schemaSuffix ? '-' + schemaSuffix : ''
+            }.json`
           ),
           schema_id: typeUri(node, hostname),
-          validation_level: isSlot ? 'SLOT' : 'CONTENT_TYPE'
+          validation_level: isSlot ? 'SLOT' : 'CONTENT_TYPE',
         })
 
         const shouldVisualize = maybeDirectiveValue<BooleanValueNode>(
           directive,
           'visualizations'
         )?.value
 
+        const iconUrl = maybeDirectiveValue<EnumValueNode>(
+          maybeDirective(node, 'icon')!,
+          'url'
+        )?.value
+
         const contentType = tfg.resource('amplience_content_type', name, {
           content_type_uri: schema.attr('schema_id'),
           label: capitalCase(node.name.value),
+          icon: iconUrl ? { size: 256, url: iconUrl } : undefined,
           status: 'ACTIVE',
           visualization:
             shouldVisualize && visualization
               ? {
                   label: 'Visualization',
                   templated_uri: maybeArg(visualization),
-                  default: true
+                  default: true,
                 }
-              : undefined
+              : undefined,
+          card: [{}], // TODO
```

Comment:
Please either remove this property or implement it :)

Reply:
will remove it, and implement it later once we get into the card topic